examples/worst_case.py

```diff
@@ -0,0 +1,319 @@
+"""
+This task evaluates a set of metrics, mostly related to worst-class performance, as described in
+(J. Bitterwolf et al., "Classifiers Should Do Well Even on Their Worst Classes", https://openreview.net/forum?id=QxIXCVYJ2WP).
+It is motivated by
+(R. Balestriero et al., "The Effects of Regularization and Data Augmentation are Class Dependent", https://arxiv.org/abs/2204.03632)
+ where the authors note that using only accuracy as a metric is not enough to evaluate
+ the performance of the classifier, as it must not be the same on all classes/groups.
+"""
```

Comment:
```suggestion
"""<Please add your title here -- this will be rendered in the docs>
This task evaluates a set of metrics, mostly related to worst-class performance, as described in
(J. Bitterwolf et al., "Classifiers Should Do Well Even on Their Worst Classes", https://openreview.net/forum?id=QxIXCVYJ2WP).
It is motivated by
(R. Balestriero et al., "The Effects of Regularization and Data Augmentation are Class Dependent", https://arxiv.org/abs/2204.03632)
 where the authors note that using only accuracy as a metric is not enough to evaluate
 the performance of the classifier, as it must not be the same on all classes/groups.
"""
```

Also, ideally format the docstring using RST syntax (e.g. right now, the links will not render properly in the documentation). Thanks :)

Reply:
Hi @stes, I've tried to incorporate all the changes - are they fine like this? :)

Reply:
As for the unit testing, @stes, yes, we could easily evaluate metrics on the subset of points, but the main time is taken by the computation of the logins currently, I think, which is still not displayed using shifthappens.config.verbose = args.verbose.